xml/System.Runtime.InteropServices/RuntimeInformation.xml

```diff
@@ -116,8 +130,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a string that indicates the name of the operating system on which the app is running.</summary>
+        <value>The name of the operating system on which the app is running.</value>
```

Comment:
NIT: I'm not sure `name` is the right word here, more like it is the description of the OS. Foro example, if running in .NET Native, we will return `Microsoft Windows` which not necessarily means that this is the name of the OS.